src/shared/version_op.c

```diff
@@ -715,6 +716,29 @@ int get_nproc() {
     }
 
     return CPU_COUNT(&set);
+    #else
+    FILE *fp;
+    char string[OS_MAXSTR];
+    char *end;
+    int cpu_cores = 0;
+
+    if (!(fp = fopen("/proc/cpuinfo", "r"))) {
+        mwarn("Unable to read cpuinfo file");
+    } else {
+        char *aux_string = NULL;
+        while (fgets(string, OS_MAXSTR, fp) != NULL){
+            if ((aux_string = strstr(string, "processor")) != NULL){
```

Comment:
I think is safer to check if the string starts with `processor`, we are not sure if it can be included in other descriptions or any other place. You don't need `aux_string` as well.
`if (!strncmp(string, "processor", 9)) {`